pe_file/pe_file.py

```diff
@@ -764,6 +765,12 @@ def get_signature_information(self, file_handle):
                     res.add_subsection(ResultSection("Unknown exception. Traceback: %s" % traceback.format_exc()))
                 self.file_res.add_section(res)
                 return
+            except SignerInfoParseError as e :
+                if str(e).startswith("SignerInfo.version must be 1"):
+                    res.add_subsection(ResultSection("Invalid SignerInfo Version"))
+                else:
+                    res.add_subsection(ResultSection("Unknown exception. Traceback: %s" % traceback.format_exc()))
```

Comment:
I think any error messages with traceback information should be logged rather than displayed to the user. Traceback adds a lot of noise to the result and normally any issue requiring traceback to debug needs the system maintainer to fix it rather than the analyst.

Reply:
right I saw that in other areas of the service the traceback was shown too . So your saying it should be changed everywhere?

Reply:
Oh, I see. It's consistent in that method. Without seeing what the traceback looks like I can't judge, but if it needs to be changed it can be in a different pull request. Your fixes don't depend on which is used.